services/contract_update.proto

```diff
@@ -109,4 +109,14 @@ message ContractUpdateTransactionBody {
      * account. Otherwise it updates the contract's auto-renew account to the referenced account.
      */
     AccountID auto_renew_account_id = 12;
+
+    /**
+     * ID of the new account to which this contract is staking. If not staked to an account, the value is null. The default value is null.
+     */
+    AccountID staked_account_id = 13;
```

Comment:
Is it possible to clear an existing `staked_account_id` or `staked_node_id`? How would that look like with a `oneof`?

Reply:
I have updated to use `oneof` with the `Int64Value` Wrapper. And in the implementation we will make sure only one filed is set.